bin/review-rot

```diff
@@ -62,13 +64,16 @@ def main(state_, value, duration, ssl_verify=None):
                 """
                 res = format_user_repo_name(data)
                 """
-                get pull requests for specified git service, username
-                and repository name
+                get pull/merge/change requests for specified git service
                 """
+                repo_name = data
```

Comment:
In that case, are you still planning to remove these lines and update the `repo_name` kwarg in the call to `git_service.request_reviews` below?